Automattic/simplenote-ios
# Add screenshots composer configuration for iPhone 8 Plus

### What it does
Adds the config setup to run the promo screenshots generation (screenshots composer?) for iPhone 8 Plus.

I've added only this one so I can get feedback on the setup itself. If everything's good. Then I can replicate for the other device sizes.

### Test
1. Run `BUNDLE_WITH=screenshots bundle install`, please do not commit the resulting change in `.bundle/config`, if any. See comment on that file for rationale.
1. Run `bundle exec fastlane take_screenshots && bundle exec fastlane create_promo_screenshots`.

If everything works, you should have six screenshots in your `fastlane/promo_screenshots/en-US/` folder exactly like these.

#### Dark Mode

`bundle exec fastlane take_screenshots mode:dark && bundle exec fastlane create_promo_screenshots`

<table>
<tr>
<td>
<img alt="iPhone 8 Plus-01" src="https://user-images.githubusercontent.com/1218433/78099130-b4dd3380-742c-11ea-90ce-0c95c2814745.png"/>
</td>
<td>
<img alt="iPhone 8 Plus-02" src="https://user-images.githubusercontent.com/1218433/78099430-962b6c80-742d-11ea-8a75-8ead54ec0a74.png" />
</td>
<td><img alt="iPhone 8 Plus-03" src="https://user-images.githubusercontent.com/1218433/78099156-c1fa2280-742c-11ea-8538-277839a376a9.png" /></td>
<td><img alt="iPhone 8 Plus-04" src="https://user-images.githubusercontent.com/1218433/78099158-c4f51300-742c-11ea-8411-8e8e3e12ecb9.png" /></td>
<td><img alt="iPhone 8 Plus-05" src="https://user-images.githubusercontent.com/1218433/78099162-c6bed680-742c-11ea-89c5-f7266133a29c.png" /></td>
<td><img alt="iPhone 8 Plus-06" src="https://user-images.githubusercontent.com/1218433/78099166-c9213080-742c-11ea-818d-2f1e3cabda31.png" /></td>
</tr>
</table>

#### Light Mode

`bundle exec fastlane take_screenshots mode:light && bundle exec fastlane create_promo_screenshots`

<table>
<tr>
<td>
<img alt="iPhone 8 Plus-01" src="https://user-images.githubusercontent.com/1218433/78100897-962d6b80-7431-11ea-96e2-0749c0462e42.png" width=300/></td>
<td><img alt="iPhone 8 Plus-02" src="https://user-images.githubusercontent.com/1218433/78100940-aba29580-7431-11ea-8b9b-8313f6da831f.png" width=300/></td>

<td><img alt="iPhone 8 Plus-03" src="https://user-images.githubusercontent.com/1218433/78100959-b9f0b180-7431-11ea-93df-9a956d622317.png" width=300/></td>

<td><img alt="iPhone 8 Plus-04" src="https://user-images.githubusercontent.com/1218433/78100963-bceba200-7431-11ea-9a1d-74b12d763942.png" width=300/></td>
<td><img alt="iPhone 8 Plus-05" src="https://user-images.githubusercontent.com/1218433/78100967-c117bf80-7431-11ea-9310-21dfd306e95d.png" width=300/></td>
<td><img alt="iPhone 8 Plus-06" src="https://user-images.githubusercontent.com/1218433/78100988-cffe7200-7431-11ea-96f3-3f9f919d401d.png" width=300/></td>
</tr>
</table>

### Review
I think only @jkmassel  has enough context to review this PR. @SylvesterWilmott I'd love your input on the screenshots output: is this what you were expecting?

### Release
These changes do not require release notes.
